### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by matfior-finn** [calogica/dbt-date#111]
### Is this a new bug in dbt-date?

- [ ] I believe this is a new bug in dbt-date
- [ ] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

When installing [dbt-expectations](https://github.com/calogica/dbt-expectations) and [dbt-date](https://github.com/calogica/dbt-date) I get an error.

### Expected Behavior

Both packages installed

### Steps To Reproduce

Have both packages at their latest version in your packages.yml file and then run dbt deps

### Relevant log output

Version error for package calogica/dbt_date: Could not find a satisfactory version from options: ['=0.9.1', '>=0.8.0', '<0.9.0']

### Environment

```markdown
- OS: MacOs Ventura
- Python: 3.9
- dbt: 1.6.1
- dbt-date:  0.9.1
```

### Which database adapter are you using with dbt?

- BigQuery

**Comment by clausherther** [comment]
Could you please post the contents of your `packages.yml` file? Sounds like you may have a typo or other issue there.

**Comment by matfior-finn** [comment]
`packages:
  - package: calogica/dbt_date
    version: 0.9.1
  - package: calogica/dbt_expectations
    version: 0.9.0
  - package: kristeligt-dagblad/dbt_ml
    version: 0.6.0`

**Comment by matfior-finn** [comment]
Not a typo, because if you check this file:
https://github.com/calogica/dbt-expectations/blob/main/packages.yml
you'll see that it needs:
```packages:
  - package: calogica/dbt_date
    version: [">=0.8.0", "<0.9.0"]
but if you try to pin date to 0.9.1 you'll get the error

**Comment by clausherther** [comment]
your issues are:
a) you only need dbt-expectations, you don't need to inlude dbt-date as well (see https://github.com/calogica/dbt-expectations#dependencies)
b) it's best to use ranges and not to pin to minor releases

```yaml
packages:
  - package: calogica/dbt_expectations
    version: [">=0.9.0", "<0.10.0"]
```

**Comment by matfior-finn** [comment]
Even if I remove the package from the file I get this:

```
15:23:55  Running with dbt=1.6.1
15:23:56  Installing calogica/dbt_expectations
15:23:56  Installed from version 0.9.0
15:23:56  Up to date!
15:23:56  Installing kristeligt-dagblad/dbt_ml
15:23:57  Installed from version 0.6.0
15:23:57  Up to date!
15:23:57  Installing calogica/dbt_date
15:23:57  Installed from version 0.8.1
15:23:57  Updated version available: 0.9.1
15:23:57  Installing dbt-labs/dbt_utils
15:23:57  Installed from version 1.1.1
15:23:57  Up to date!
15:23:57  
15:23:57  Updates available for packages: ['calogica/dbt_date']                 
Update your versions in packages.yml, then run dbt deps
```

Because of the above requirement.

**Comment by clausherther** [comment]
Yup, that's ok, that's as intended and not an error but a warning. `dbt-expectations` isn't ready yet to use the latest `dbt-date`, which only includes updates for spark.
See https://github.com/calogica/dbt-expectations/issues/275#issuecomment-1689978410
